Dockerfile

```diff
@@ -11,14 +11,14 @@
 #     Switch to official CentOS Stream from quay.io
 #     https://www.linux.org/threads/centos-announce-centos-stream-container-images-available-on-quay-io.33339/
 
-ARG BASE_IMAGE=quay.io/centos/centos:stream9
+ARG BASE_IMAGE=docker.io/rockylinux/8.6
```

Comment:
This should be `docker.io/rockylinux:8.6` (not `/8.6`).

Reply:
Thanks!